Backend/app/dashboard/CommonAPIs/spec/RiderPlatform/AppManagement/API/Tickets.yaml

```diff
@@ -0,0 +1,91 @@
+imports:
+  APISuccess: Kernel.Types.APISuccess
+  TicketServiceResp: API.Types.UI.TicketService
+  TicketBookingUpdateSeatsReq: API.Types.UI.TicketService
+  TicketBookingCancelReq: API.Types.UI.TicketService
+  TicketServiceCancelReq: API.Types.UI.TicketService
+  TicketBookingDetails: API.Types.UI.TicketService
+  TicketServiceVerificationResp: API.Types.UI.TicketService
+  TicketService: Domain.Types.TicketService
+  TicketBookingService: Domain.Types.TicketBookingService
+  TicketPlace: Domain.Types.TicketPlace
+  TicketBooking: Domain.Types.TicketBooking
+
+importPackageOverrides:
+  API.Types.UI.TicketService: "rider-app"
+  Domain.Types.TicketService: "rider-app"
+  Domain.Types.TicketBookingService: "rider-app"
+  Domain.Types.TicketPlace: "rider-app"
+  Domain.Types.TicketBooking: "rider-app"
+
+module: Tickets
+apiPrefix: ""
+
+apis:
+  - POST: # VerifyBookingDetailsAPI
+      endpoint: /tickets/{personServiceId}/{ticketBookingShortId}/verify
```

Comment:
in manual version it was:

```
type API =
  "tickets"
    :> VerifyBookingDetailsAPI
    :<|> GetServicesAPI
    :<|> UpdateSeatManagementAPI
    :<|> GetTicketPlacesAPI
    :<|> CancelTicketBookingServiceAPI
    :<|> CancelTicketServiceAPI
    :<|> GetTicketBookingDetails


```

as `:>` operator has higher priority then `:<|>`, so "tickets" api prefix applied only to first api `VerifyBookingDetailsAPI`

```
data (path :: k) :> (a :: [Type] infixr 4

data a :<|> b infixr 3
```